libs/utils/analysis/tasks_analysis.py

```diff
@@ -186,6 +186,42 @@ def _dfg_rt_tasks(self, min_prio=100):
 
         return rt_tasks
 
+    def _dfg_task_lt_events(self):
```

Comment:
Should not `df` be initialised to None?

Reply:
Moreover: I think `task_load_events` was a better name... actually, to be more scheduler aligned, what about `sched_task_load`?